system/bootstrap.php

```diff
@@ -77,7 +78,7 @@
 	require_once APPPATH . 'Config/Constants.php';
 }
 
-// Let's see if an app/Common.php file exists
+// Require app/Common.php file if exists.
 if (file_exists(APPPATH . 'Common.php'))
```

Comment:
While you're here, I've read that `is_file()` is way faster than `file_exists` - and also technically more correct in this case.